src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,443 @@
+meat of the blowfish is a special taste experience. If prepared incorrectly, however, it can be
+fatal. The powerful APIs of Project Fugu are extremely exciting for developers. However, they can
+affect the security and privacy of the user. Therefore, the Fugu team pays special attention to
+these issues: For instance, new interfaces require the website to be sent over a secure connection
+(HTTPS). Some of them require a user gesture, such as a click or key press, to prevent fraud. Other
+capabilities require explicit permission by the user. Developers can use all APIs as a progressive
+enhancement: By feature detecting the APIs, applications won't break in browsers lacking support for
+those capabilities. In browsers that support them, users can get a better user experience. This way,
+web apps [progressively enhance](https://web.dev/progressively-enhance-your-pwa/) according to the
+user's particular browser.
+
+This chapter gives an overview of various modern web APIs, and the state of web capabilities in 2020
+based on usage data by the HTTP Archive. Due to technical limitations, the HTTP Archive only has
+data available for APIs that require neither permission, nor a user gesture. Where no data is
+available, the percentage of page loads in Google Chrome according to
+[Chrome Platform Status](https://chromestatus.com/metrics/feature/timeline/popularity) will be shown
+instead. Since some interfaces are brand-new and their usage is still very low, the statistics are
+not necessarily meaningful. However, in many cases trends can still be read from the data. Unless
+otherwise noted, the APIs are only available in Chromium-based browsers, and their specifications
+are in the early stages of standardization.
+
+## Async Clipboard API
+
+With the help of the `document.execCommand()` method, websites could already access the user's
+clipboard. However, this approach is somewhat restricted, as the API is synchronous (making it
+difficult to process clipboard items), and it can only interact with selected text in the DOM. This
+is where the [Async Clipboard API](https://web.dev/async-clipboard/)
+([W3C Working Draft](https://www.w3.org/TR/clipboard-apis/#async-clipboard-api)) comes in. This new
+API is not only asynchronous, meaning it doesn't block the page for large chunks of data or waiting
+for a permission to be granted, but also allows for images to be copied to or pasted from the
+clipboard in supported browsers such as Chrome, Edge, and Safari.
+
+### Read Access
+
+The Async Clipboard API provides two methods for reading content from the clipboard: A shorthand
+method for plain text, called `navigator.clipboard.readText()`, and a method for arbitrary data,
+called `navigator.clipboard.read()`. Currently, most browsers only support HTML content and PNG
+images as additional data formats. Due to privacy reasons, reading from the clipboard always
+requires the user's consent.
+
+(IMAGE: Async Clipboard API)
```

Comment:
For any visual or tabular figures to be embedded in the chapter, please refer to the [Figures Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) in the wiki for info on the expected markdown syntax to use and how to add metadata.

Reply:
Done, left the description at "TODO" until we decide which data to choose